src/components/scenes/existingAccout/SecurityAlertsScene.tsx

```diff
@@ -158,32 +188,52 @@ export class SecurityAlertsSceneComponent extends React.Component<
     const styles = getStyles(theme)
 
     return pendingVouchers.map(voucher => (
-      <View style={styles.tile} key={voucher.voucherId}>
-        <MessageText>
-          {voucher.deviceDescription != null
-            ? lstrings.alert_scene_device + voucher.deviceDescription + '\n'
-            : null}
-          {lstrings.alert_scene_ip + voucher.ipDescription + '\n'}
-          {lstrings.alert_scene_date + toLocalTime(voucher.created)}
-        </MessageText>
-        <MessageText>
-          {lstrings.alert_scene_reset_date + toLocalTime(voucher.activates)}
-        </MessageText>
+      <EdgeCard key={voucher.voucherId}>
+        <View style={styles.textBlock}>
+          {voucher.deviceDescription != null ? (
+            <EdgeText>
+              {lstrings.alert_scene_device + voucher.deviceDescription}
+            </EdgeText>
+          ) : null}
+          <EdgeText>{lstrings.alert_scene_ip + voucher.ipDescription}</EdgeText>
+          <EdgeText>
+            {lstrings.alert_scene_date + toLocalTime(voucher.created)}
+          </EdgeText>
+        </View>
+
+        <View style={styles.textBlock}>
+          <EdgeText numberOfLines={2}>
+            <WarningText>
+              {lstrings.alert_scene_reset_date + toLocalTime(voucher.activates)}
+            </WarningText>
+          </EdgeText>
+        </View>
         {spinVoucher[voucher.voucherId] ? (
           <ActivityIndicator
             color={theme.primaryButton[0]}
             style={styles.cardSpinner}
           />
         ) : (
-          <LinkRow
-            label={lstrings.alert_scene_approve}
-            onPress={() => this.handleApproveVoucher(voucher.voucherId)}
-            renderIcon={theme => (
-              <AntDesignIcon name="check" size={theme.rem(1)} />
-            )}
-          />
+          // TODO: Codify this into a ButtonView variant if we need this
+          // combination elsewhere. Hard-coded because it's assumed to be a
+          // weird one-off situation.
+          <StyledButtonContainer layout="row">
+            <EdgeButton
+              label={lstrings.allow}
+              onPress={this.handleApproveVoucher(voucher)}
+              type="secondary"
```

Comment:
We did go back and forth a lot during spec discussion on this. Ultimately we went with this, because a deny/block/delete/no type button is generally expected to be red.

In this situation, "allow" in itself is the opposite of a destructive operation, but at the same time, it's dangerous.

The compromise was to add the exclamation marks to convey danger on the secondary "Allow" in the following modal.

Now that you mention this, 'danger' is probably just a bad choice of naming, just like how our legacy 'escape' was also inaccurate. I'll change it.

Reply:
I absolutely do not agree with this decision. Who is the "we" in this discussion? Madison? Michael? Paul?

Since "Deny" is the ONLY safe button to press, marking it as "danger" is exactly the opposite of what we want. This completely negates the whole reason we approved this task for the sprint, and is the exact opposite of what we wanted. The support department asked for this feature specifically because customers had hit the wrong button and lost all their funds.

Valid solutions would include either swapping the danger marker, vetoing Madison or whoever was in your discussion, or making both buttons equal. I will not approve this PR with "Deny" marked dangerous, sorry.

Reply:
Continuing with the design post discussion because the strongest point of evidence that this aligns with user expectations is that google/apple/etc have the same design:

Red denotes the destructive (red) "Deny" action, and they go further to even make the "Allow/Yes this was me" the primary action